nextcord/enums.py

```diff
@@ -2037,6 +2037,20 @@ class ForumLayoutType(IntEnum):
     """Display posts as a collection of posts with images, this is more image focused."""
 
 
+class InviteType(IntEnum):
+    """Represents the type of an invite.
+
+    .. versionadded:: 3.10
+    """
+
+    guild = 0
+    """The invite is for a guild."""
+    group_dm = 1
+    """The invite is for a group DM"""
```

Comment:
Gotta correct you with one dot 😉

```suggestion
    """The invite is for a group DM."""
```